### PR TITLE
chore/update-dashboard-screenshots

### PR DESCRIPTION
**Pull request by doralchan**
Before & After:
Screen Shot 2022-01-20 at 2 38 56 PM
Screen Shot 2022-01-20 at 2 39 05 PM

### CODE REVIEW

**Review by wmak — APPROVED** [review]
LGTM, should wait until we GA the Library & Template features before we merge

**Review by imatwawana — APPROVED** [review]

**Pull request merged**
